paddle/fluid/operators/detail/grpc_server.cc

```diff
@@ -418,15 +396,29 @@ void AsyncGRPCServer::HandleRequest(
   }
 }
 
-void AsyncGRPCServer::WaitCond(int cond) {
+void AsyncGRPCServer::RegisterBarrier(const std::string& rpc_name,
+                                      int cond_id) {
   std::unique_lock<std::mutex> lock(this->barrier_mutex_);
```

Comment:
do we really need a map? It seems we only have a few rpc methods. A enum is enough?